### PR TITLE
switch sign job to use eclipse codesign cert

### PR DESCRIPTION
**Pull request by gdams**
depends on https://github.com/adoptium/temurin-build/pull/2619 being merged first.

### CODE REVIEW

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -395,36 +395,28 @@ class Build {
     */
     def sign(VersionInfo versionInfo) {
         // Sign and archive jobs if needed
-        // TODO: This version info check needs to be updated when the notarization fix gets applied to other versions.
         if (
-            buildConfig.TARGET_OS == "windows" ||
-        (buildConfig.TARGET_OS == "mac" && versionInfo.major == 8 && buildConfig.VARIANT != "openj9") || (buildConfig.TARGET_OS == "mac" && versionInfo.major == 13)
+            buildConfig.TARGET_OS == "windows" || (buildConfig.TARGET_OS == "mac" && versionInfo.major == 8)
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy:399]: logic is changed from above - is that really correct (excluding the removal of openj9)?

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 2]
```diff
@@ -395,36 +395,28 @@ class Build {
...
                 }
 
                 def params = [
                         context.string(name: 'UPSTREAM_JOB_NUMBER', value: "${env.BUILD_NUMBER}"),
                         context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
                         context.string(name: 'OPERATING_SYSTEM', value: "${buildConfig.TARGET_OS}"),
                         context.string(name: 'VERSION', value: "${versionInfo.major}"),
+                        context.string(name: 'SIGN_TOOL', value: "eclipse"),
                         context.string(name: 'FILTER', value: "${filter}"),
-                        context.string(name: 'CERTIFICATE', value: "${certificate}"),
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy:427]: What uses CERTIFICATE Downstream?

**Review by karianna — CHANGES_REQUESTED** [review]

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 1 above)
Comment (gdams) [reply pipelines/build/common/openjdk_build_pipeline.groovy:399]: yes, we now want to use the previous signing method for jdk8 (hotspot and J9)

**Review by karianna — APPROVED** [review]
LGTM

**Comment by gdams** [comment]
run tests

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 3]
```diff
@@ -395,36 +395,28 @@ class Build {
     */
```
Comment (AdamBrousseau) [inline pipelines/build/common/openjdk_build_pipeline.groovy:395]: This comment needs to be updated to reflect the new logic below.

**Pull request merged**